src/utilities.h

```diff
@@ -555,4 +571,9 @@ static inline std::string trimCopy(std::string s)
     return s;
 }
 
+// KRM need docstrings.
```

Comment:
yes.  :)

Reply:
These are slightly different in the PR which uses them more #639 (we've now dropped testing/adding mathml ids, for example).  Should I merge that one with this one first?  It seems odd to change it in two places (which will have knock-on effects throughout the printer too).

Reply:
Will be done in the other PR instead.